.github/workflows/integration_tests.yml

```diff
@@ -482,6 +487,13 @@ jobs:
         timeout-minutes: 5
         run: echo "y" | keptn uninstall -n ${KEPTN_NAMESPACE}
 
+      - name: Uninstall remote execution plane
+        timeout-minutes: 5
+        if: env.REMOTE_EXECUTION_PLANE == 'true'
+        run: |
+          kubectl delete namespace keptn-helm-service
+          kubectl delete namespace keptn-jmeter-service
+
```

Comment:
I like that we have this here. Does it make sense to also have the install and verification step outside, right before the test is executed?